src/index.js

```diff
@@ -32,4 +32,6 @@ document.addEventListener('click', (event) => {
 clearButton.addEventListener('click', () => {
   Tasklist.createList();
   window.location.reload();
+  const filteredList = list.filter((obj) => obj.complete !== true);
+  localStorage.setItem('listStorage', JSON.stringify(filteredList));
```

Comment:
[OPTIONAL] After clicking on the "Clear all completed" button, all the indexes of remained tasks in the local storage should be updated, so they represent the current list order and are unique.